app/src/main/java/com/yml/womensafety/navigationdrawer/TipsForWomenSafetyFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.yml.womensafety.navigationdrawer
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.yml.womensafety.TipsForWomenSafetyAdapter
+import com.yml.womensafety.R
+
+class TipsForWomenSafetyFragment : Fragment(R.layout.fragment_tips_for_women_safety) {
+
+    private val tipsTextList = ArrayList<SafetyTips>()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val recyclerView = view.findViewById(R.id.recyclerView) as RecyclerView
+        recyclerView.layoutManager = LinearLayoutManager(view.context)
+
+
+        displaySafetyTips()
+
+        val adapter = TipsForWomenSafetyAdapter(tipsTextList)
+        recyclerView.adapter = adapter
+    }
+
+    private fun displaySafetyTips() {
+        tipsTextList.add(SafetyTips("If walking by yourself, get off your cell phone. Chatting can distract you, " +
```

Comment:
Avoid using hard coded strings

Reply:
We are following MVVM architecture. You should build data in repo layer.

Reply:
Done